### DISCUSSION TITLE
Sending emails not possible with O365

### DISCUSSION THREAD

**Discussion by idemus** [kimai/kimai#4915] [General] [answered]
Hi,
I've tried to setup email notifications within .env file with following entry:
MAILER_URL=smtp://smtp.office365.com:587?encryption=tls&auth_mode=login&username=xxxx@idemus.com&password=yyyy

Even that the user is a full licensed user in O365 I get following log error message in Kimai:
[app] CRITICAL: Expected response code "250" but got code "530", with message "530 5.7.57 Client not authenticated to send mail.

Does someone has an idea how to setup email settings for O365 in Kimai?

Best regards
Klaus

**Comment by kevinpapst** [comment, marked as answer]
In URLs username and password usually belong to the front:

```
smtp://xxxx@idemus.com:yyyy@smtp.office365.com:587?encryption=tls
```

But I am not sure if that is enough...

**Reply by idemus** [reply]
Thank you! That did the job.
